de_workloads/ingest/Ingest_AzureSql_Example/config/ingest_sources/Ingest_AzureSql_Example.json

```diff
@@ -1,40 +1,51 @@
     "data_source_type": "azure_sql",
     "enabled": true,
     "ingest_entities": [
         {
             "version": 1,
-            "display_name": "SalesLT.Product",
+            "display_name": "movies.keywords",
             "enabled": true,
-            "schema": "SalesLT",
-            "table": "Product",
-            "columns": "*",
+            "schema": "movies",
+            "table": "keywords",
+            "columns": "[id], [keywords]",
             "load_type": "full",
             "delta_date_column": null,
             "delta_upsert_key": null
         },
         {
             "version": 1,
-            "display_name": "SalesLT.SalesOrderHeader",
+            "display_name": "movies.links",
             "enabled": true,
-            "schema": "SalesLT",
-            "table": "SalesOrderHeader",
-            "columns": "SalesOrderID, RevisionNumber, OrderDate, DueDate, ShipDate, Status, OnlineOrderFlag, SalesOrderNumber, PurchaseOrderNumber, AccountNumber, CustomerID, ShipToAddressID, BillToAddressID, ShipMethod, CreditCardApprovalCode, SubTotal, TaxAmt, Freight, TotalDue, Comment, rowguid, ModifiedDate",
-            "load_type": "delta",
-            "delta_date_column": "ModifiedDate",
-            "delta_upsert_key": "SalesOrderID"
+            "schema": "movies",
+            "table": "links",
+            "columns": "[movieId], [imdbId], [tmdbId]",
+            "load_type": "full",
+            "delta_date_column": null,
+            "delta_upsert_key": null
+        },
+        {
+            "version": 1,
+            "display_name": "movies.movies_metadata",
+            "enabled": true,
+            "schema": "movies",
+            "table": "movies_metadata",
+            "columns": "[adult], [belongs_to_collection], [budget], [genres], [homepage], [id], [imdb_id], [original_language], [original_title], [overview], [popularity], [poster_path], [production_companies], [production_countries], [release_date], [revenue], [runtime], [spoken_languages], [status], [tagline], [title], [video], [vote_average], [vote_count]",
+            "load_type": "full",
+            "delta_date_column": null,
+            "delta_upsert_key": null
         },
         {
             "version": 1,
-            "display_name": "SalesLT.SalesOrderDetailID",
+            "display_name": "movies.ratings_small",
             "enabled": true,
-            "schema": "SalesLT",
-            "table": "SalesOrderDetail",
-            "columns": "SalesOrderID, SalesOrderDetailID, OrderQty, ProductID, UnitPrice, UnitPriceDiscount, LineTotal, rowguid, ModifiedDate",
+            "schema": "movies",
+            "table": "ratings_small",
+            "columns": "[userId], [movieId], [rating], [timestamp]",
             "load_type": "delta",
-            "delta_date_column": "ModifiedDate",
-            "delta_upsert_key": "SalesOrderID, SalesOrderDetailID"
+            "delta_date_column": "DATEADD(MINUTE,[timestamp]/60,'1970-01-01 00:00:00.0')",
```

Comment:
this feels too complicated, could you try using [dateTimeFromEpoch](https://learn.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions-date#datetimefromepoch)?

Reply:
The reason is that this needs to run dynamically on the data source itself, so this is T-SQL and will get used in the query, e.g.
```
SELECT [userId], [movieId], [rating], [timestamp]
FROM movies.ratings_small
WHERE DATEADD(MINUTE,[timestamp]/60,'1970-01-01 00:00:00.0') >= '2010-01-01'
AND DATEADD(MINUTE,[timestamp]/60,'1970-01-01 00:00:00.0') < '2010-01-31';
```

Reply:
OK, I understand, looks like `DATEADD` is the way then. But do we really need to do this extra calculations for minutes? Why not simply `DATEADD(SECOND,[timestamp],'19700101')`?

Reply:
Good point, updated